src/main/java/com/example/music/backend/playlist/controller/PlaylistController.java

```diff
@@ -0,0 +1,73 @@
+package com.example.music.backend.playlist.controller;
+
+import com.example.music.backend.playlist.dto.PlaylistDto;
+import com.example.music.backend.playlist.response.PlaylistResponse;
+import com.example.music.backend.playlist.service.PlaylistService;
+import com.example.music.backend.user.domain.User;
+import com.example.music.backend.user.service.UserService;
+import com.example.music.backend.video.dto.YoutubeVideoDto;
+import com.example.music.backend.video.response.YoutubeVideoResponse;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/playlists")
+public class PlaylistController {
+
+    private final PlaylistService playlistService;
+    private final UserService userService;
+
+    @PutMapping(value = "/{playlistId}", produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<YoutubeVideoResponse>> addVideoToPlaylist(
+            @PathVariable("playlistId") Long playlistId,
+            @RequestBody YoutubeVideoDto dto) {
```

Comment:
In DELETE you're using just String, but here you're using Dto. Why?

Reply:
Also replaced with @Requestbody Map <String, String>, but what's the point? To work with one parameter, I need to write a custom annotation, but why should I, if I can use VideoDto, in which there is one parameter?

Reply:
Yeah, and I expected that you would use VideoDto in both places...
 Why did you decide to use Map<String, String>?

Reply:
The Dto is back